src/client/pages/Offer/Checkout/index.tsx

```diff
@@ -441,60 +458,66 @@ export const Checkout = ({
 
   return (
     <>
-      <OuterWrapper visibilityState={visibilityState}>
-        <ScrollWrapper
-          ref={scrollWrapper as React.MutableRefObject<HTMLDivElement | null>}
-          windowHeight={windowInnerHeight}
-        >
-          <InnerWrapper>
-            <CloseButton onClick={onClose} />
-            <Section>
-              <StyledCampaignBadge
-                quoteCartId={quoteCartId}
-                isNorwegianBundle={isNorwegianBundle(offerData)}
-              >
-                <DiscountTag offerData={offerData} />
-              </StyledCampaignBadge>
-              <Heading>{textKeys.CHECKOUT_HEADING()}</Heading>
-              <PriceBreakdown
-                offerData={offerData}
-                showTotal={true}
-                isLoading={isBundleCreationInProgress}
-              />
-              <CheckoutDetailsForm formikProps={formik} />
-              <StartDateWrapper>
-                <StartDateLabel>
-                  {textKeys.SIDEBAR_STARTDATE_CELL_LABEL()}
-                </StartDateLabel>
-                <StartDate quoteCartId={quoteCartId} offerData={offerData} />
-              </StartDateWrapper>
-              {isUpsellCardVisible && (
-                <UpsellCard
-                  quoteBundleVariants={quoteBundleVariants}
-                  selectedQuoteBundleVariant={selectedQuoteBundleVariant}
-                  onUpsellAccepted={onUpsellAccepted}
+      <>
```

Comment:
Wouldn't be the same as following? 🤔 

```
<>
  <OuterWrapper>
    ...
  </OuterWrapper>
  <Backdrop visibilityState={visibilityState} onClick={onClose} />
  <SignFailModal isVisible={isShowingFailModal} onClose={() => setIsShowingFailModal(false)} />
</>
```

Reply:
I copied the markup from the old offer page. I can give your suggestion a try and see if it works

Reply:
Fixed :)